src/webui/webapplication.h

```diff
@@ -52,7 +52,7 @@
 #include "base/utils/version.h"
 #include "api/isessionmanager.h"
 
-inline const Utils::Version<3, 2> API_VERSION {2, 9, 6};
+inline const Utils::Version<3, 2> API_VERSION {2, 10, 0};
```

Comment:
I increased the second number so that from now on the API version from the `master` differs from the API versions of other branches. Otherwise, we may get problems/conflicts when backporting API changes.